packages/core/admin/admin/src/content-manager/components/RepeatableComponent/index.js

```diff
@@ -50,6 +50,37 @@ const RepeatableComponent = ({
     [componentUid, getComponentLayout]
   );
 
+  const search = useQuery();
+
+  /**
+   * Get the temp key of the component that has the field that is currently focussed
+   * as defined by the `field` query param. We can then force this specific component
+   * to be in it's "open" state.
+   */
+  const componentTmpKeyWithFocussedField = useMemo(() => {
+    if (search.has('field')) {
+      const field = search.get('field');
+
+      // eslint-disable-next-line no-unused-vars
+      let [_, path] = field.split(`${name}.`);
```

Comment:
```suggestion
      let [, path] = field.split(`${name}.`);
```

should work too without eslint ignore :)

Reply:
You can also remove the eslint ignore comment now :)